services/app-api/utils/time/time.ts

```diff
@@ -80,7 +80,7 @@ export const calculateDueDate = (
   let date = new Date();
   if (reportType === ReportType.WP) {
     reportPeriod === 1
-      ? (date = new Date(currentYear, 4, 1))
+      ? (date = new Date(currentYear, 8, 1))
```

Comment:
Small suggestion: maybe we can add a code comment above this if statement about how this date should be reverted back to month index 4 after the 2024 reporting periods...I am paranoid about someone forgetting this!

Reply:
Good point. Ill update it